### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by shuvashish76** [thePlebDev/Clicker#1566]
Hello, would you consider publishing the app on F-Droid please?

**Comment by thePlebDev** [comment]
@shuvashish76 Yes I can!! However, it will be two weeks until I start a F-Droid publishing. This week is focusing on optimization and next week is focused on discord integration(help mods better communicate)
-  If you have any feature requests please let me know!!
- Also, [HERE](https://www.reddit.com/r/modderzz/comments/1e1m10d/example_of_enhanced_modview_with_chat_integration/) is a UI demonstration of my apps enhanced modView

**Comment by shuvashish76** [comment]
ThankYou. Documentation for the inclusion : https://gitlab.com/fdroid/fdroiddata/blob/master/CONTRIBUTING.md
